🚀 5.0.0 (#280)
borisbu/OctoRelay: CHANGELOG.md

# Changelog

## Version 5

### 5.0.0

- **Breaking changes**:
  - Minimum version of Python supported: 3.9;
- Feature: Supporting Raspberry Pi 5:

>   - Using `gpiozero` with two `lgpio` and `RPi.GPIO` factories depending on your hardware;

Author: ```suggestion
  - Using `gpiozero` with `lgpio` or `RPi.GPIO` depending on your hardware;
```
